dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -106,6 +106,17 @@ isoLevel.SERIALIZABLE.desc=Dirty reads, non-repeatable reads and phantom reads a
 isoLevel.SNAPSHOT.desc=Snapshot isolation for Microsoft SQL Server JDBC Driver and DataDirect Connect for JDBC driver.
 isoLevel.NONE.desc=Indicates that the JDBC driver does not support transactions.
 
+# Identify Exception
+identifyException=Identify exception as
+identifyException.desc=Map a specific sqlCode or sqlState to an action such as a stale connection.
+
+identifyException.sqlState=SQL State
+identifyException.sqlState.desc=A string that follows either the XOPEN SQLstate conventions or the SQL:2003 conventions.
+identifyException.errorCode=SQL Error Code
+identifyException.errorCode.desc=An integer error code specific to the backend database. Normally this will be the actual error code returned by the underlying database.
+identifyException.as=Identify exception as
+identifyException.as.desc=The target mapping for the sqlCode or sqlState. Allowed values are: None (removes a mapping), or StaleConnection (connections that are determined to be stale will be evicted from the connection pool once closed)
```

Comment:
Allowed values are: None (removes a mapping), or StaleConnection (connections that are determined to be stale will be evicted from the connection pool once closed)
 --->
 Allowed values are: None, which removes a mapping, or StaleConnection. Connections that are determined to be stale are evicted from the connection pool once they are closed.